src/hardware/mixer.cpp

```diff
@@ -206,12 +212,15 @@ struct MixerSettings {
 	bool do_chorus        = false;
 
 	bool is_manually_muted = false;
+
+	bool chipmunk_mode = false;
```

Comment:
Amusing, but maybe call it `fast_forward_mode` 😆

Reply:
Renamed